### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by MichiBerger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
-
 {
   "name": "jokes-app",
```
Comment (celinevo) [inline package.json:2]: The name is probably from the copied files ;)

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,69 @@
...
 function App() {
+  const [phrases, setPhrases] = useState(PhraseData);
+
   return (
-    <div>
-      <PhraseCardList />
-    </div>
+    <AppGrid>
+      <Header>
+        <h1>header</h1>
```
Comment (celinevo) [inline src/App.js:15]: I assume the `header` text in your `h1` is kind of a placeholder?

File: src/components/EmptyPhraseMessage.spec.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+import { MemoryRouter as Router } from 'react-router-dom';
+import EmptyPhraseMessage from './EmptyPhraseMessage.js';
+
+describe('EmptyPhraseMessage', () => {
+  it('renders a title, a message text and link', () => {
+    render(
+      <Router>
+        <EmptyPhraseMessage
+          titleText="Upps...da fehlt noch was!"
+          emptyPhrasetext="Gehe zurück und markiere deine Lieblingsprüche einfach durch anklicken auf das Herzsymbol!"
```
Comment (celinevo) [inline src/components/EmptyPhraseMessage.spec.js:11]: I am not really the greatest in grammar, I am pretty sure `durch anklicken auf das Herzsymbol` is incorrect. Try `durch Anklicken des Herzsybols` or `durch Klicken auf das Herzsymbol`.

File: src/components/EmptyPhraseMessage.spec.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        />
+      </Router>
+    );
+
+    const title = screen.getByRole('heading', { level: 2 });
+    expect(title).toBeInTheDocument();
+
+    const message = screen.getByText(
+      /Gehe zurück und markiere deine Lieblingsprüche einfach durch anklicken auf das Herzsymbol!/i
```
Comment (celinevo) [inline src/components/EmptyPhraseMessage.spec.js:20]: Same as above :D

File: src/components/pages/FavoritePhrases.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+import PhraseCard from '../PhraseCard.js';
+import EmptyPhraseMessage from '../EmptyPhraseMessage.js';
+
+export default function FavoritePhrases({ onIconClick, phrases }) {
+  const emptyPhrases = phrases.filter(phrase => phrase.isBookmarked);
+
+  const emptyPhrasesMessage =
+    emptyPhrases.length === 0 ? (
+      <EmptyPhraseMessage
+        emptyPhrasetext="Gehe zurück und markiere deine Lieblingsprüche einfach durch anklicken auf das Herzsymbol!"
```
Comment (celinevo) [inline src/components/pages/FavoritePhrases.js:11]: Same as above

**Review by celinevo — COMMENTED** [review]
Overall great code! Really good to read. I only found small things ;)

File: package.json (hunk 1 above)
Comment (MichiBerger) [reply package.json:2]: 👍 thx

**Pull request merged**